lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +107,26 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if (not isinstance(tokenized_document, list) or
+                not all(isinstance(text, str) for text in tokenized_document) or
+                not tokenized_document):
+            raise ValueError(f'Incorrect value: "{tokenized_document}" is not a list or empty.')
+
+        if not self._vocabulary or not self._corpus:
+            return ()
+        idf_document = calculate_idf(self._vocabulary, self._corpus)
+
+        avg_doc_len = sum(len(doc) for doc in self._corpus) / len(self._corpus)
+        doc_len = len(tokenized_document)
+        bm25_dict = calculate_bm25(self._vocabulary, tokenized_document,
+                                   idf_document, 1.5, 0.75, avg_doc_len, doc_len)
```

Comment:
do not set default arguments - use named arguments

Reply:
I wonder whether I should get new values for k1 and b or just delete them because of their defalt value